contribs/gmf/src/controllers/abstractmobile.js

```diff
@@ -97,6 +107,22 @@ gmf.AbstractMobileController = function(config, $scope, $injector) {
   goog.base(
       this, config, $scope, $injector);
 
+  // Automatically close the nav when a measure tool becomes active
+  $scope.$watch(goog.bind(function() {
```

Comment:
This should be discussed with @pgiraud and @tsauerwein.
But should be good for now.